.spyproject/codestyle.ini

```diff
@@ -0,0 +1,6 @@
+[codestyle]
```

Comment:
What is it? Please remove all non-relevant files (all from `.spyproject` folder).